tests/web3js/config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     web3: web3,
     eoa: web3.eth.accounts.privateKeyToAccount('0xf6d5333177711e562cabf1f311916196ee6ffc2a07966d9d4628094073bd5442'), // eoa is 0xfacf71692421039876a5bb4f10ef7a439d8ef61e
     fundedAmount: 5.0,
-    startBlockHeight: 3n, // start block height after setup accounts
+    startBlockHeight: 4n, // start block height after setup accounts
```

Comment:
**Note:** This one increased because when setting up integration tests, we also create a new Flow account, with multiple account keys, to be used from the `Keystore`. This also causes 1 more Flow block production, and if Flow block also creates an EVM block, which can be empty.